workflow/daily_municipality.Rmd

```diff
@@ -0,0 +1,128 @@
+---
+pagetitle: Daily Municipality Update
+output: html_document
+runtime: shiny
+css: daily_municipality.css
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+require(tidyverse)
+require(data.table)
+require(xtable)
+
+setwd('../')
+
+dat.totals.color <- read.csv("data/municipality-totals-color.csv")
+dat.totals.growth <- read.csv("data/municipality-totals-growth.csv")
+dat.today.wide <- read.csv("data/municipality-today-detailed.csv") %>%
+  arrange(municipality)
+
+print_municipality_data <- function(data) {
+  template <- "%s (%+d) (*w: %+d *%s)"
+  data.printable <- transmute(data,
+    current = current,
+    color = color_incl_new,
+    text = sprintf(template, municipality, increase_1d, increase_7d, growth)
+  )
+
+  table <- xtable(data.printable, auto = TRUE)
+  align(table) <- "llll"
+
+  print( table, 
+    type = "html", 
+    html.table.attributes = 'border=0', 
+    include.colnames = FALSE, 
+    include.rownames = FALSE
+  )
+}
+
+```
+
+### Legenda
+✅ geen geconstateerde besmettingen  
+💥 nieuwe besmettingen in een groene gemeente  
+🟡  < 5 / 100.000 inwoners / 7 dagen  
+🟧  < 50 / 100.000 inwoners / 7 dagen  
+🛑  >= 50 / 100.000 inwoners / 7 dagen  
+
+⬆️ ⬆️ Fors stijgende trend ( > 100% )  
+⬆️ Stijgende trend ( 0 - 100% )  
+⬇️ Dalende trend   
+
+Bovenstaande zijn berekend op laatste  
+7 dagen t.o.v. de 7 dagen daarvoor  
+
+### Voorbeeld:
+*17	🛑	Montfoort (+5) ( w: +10 ⬆️⬆️ )*
+      
++17 sinds 1 juli     
++5 sinds gisteren     
++10 sinds 7 dagen ( 	22-08-2020 )  
+Wat meer is dan de +1 in de 7 dagen ervoor  
+13.901 inwoners maakt dat	🛑	71,9 / 100.000 / 7d 
+
+### Landelijk:
+19.301 🟧 totaal sinds 1 juli (+501) ( w: +3.535 ⬇️ )   
+142 / 355 gemeentes met nieuwe besmettingen (w: 299 )   
+116 	Gemeente Onbekend (+2)   
```

Comment:
@mzelst waar kan ik deze data het beste vandaan plukken? Of zal ik de "totaal" en "onbekend" regel toevoegen aan de municipality sheet?